index.html

```diff
@@ -22,7 +22,7 @@ <h1 id="title">Dr. Norman Borlaug</h1>
             hunger.
           </p>
         </div>
-        <div id="tribute-info">
+        <div id="tribute-detail">
```

Comment:
Nice work. I suggest you use "class" instead of id. It will make your work easier and more professional.